package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@snapview/sunrise-dom",
-    "version": "1.0.0",
+    "version": "1.1.0",
```

Comment:
In general, I think version bumps  should not be part of the branch about to merge. Imagine this is a big MR/PR with some time consuming back and forth discussions. If in the meantime would be a small PR with a fix and a new version/release to make this fix available, version numbers might conflict.

I would suggest to keep version bumps out of PR/MR and only increase version and add tagged commit in master/main after features were merged.

Reply:
Fully agree here, @pabra.  
Would you mind keeping it here this time?